### ISSUE TITLE
ToRunOnceAt and AndEvery new issue

### ISSUE THREAD

**Issue by AlexAlexGoTO** [fluentscheduler/FluentScheduler#86]
My code for example

`Schedule<CreateRideJob>().ToRunOnceAt(DateTime.Now.AddMinutes(5)).AndEvery(1).Minutes();`

my app started and after first minute program create my task and after every 1 minute. at 5 minute program create 2 tasks in one time.

any solutions ?

I need Delay 5 minutes for first job creating

**Comment by tallesl** [comment]
I'm sorry but I didn't quite get it what you're saying. Do you want to run a job every minute but delay its start for 5 minutes?

There's `DelayFor` for that.

```cs
Schedule<CreateRideJob>().ToRunEvery(1).Minutes().DelayFor(5).Minutes();
```

**Comment by AlexAlexGoTO** [comment]
Nice,thanks, but if I need to  create job in a specific date. For example I wanna create job at 2016/8/8 and after that every 1 minute.

I thought can do that 
`Schedule<CreateRideJob>().ToRunOnceAt(2016/8/8).AndEvery(1).Minutes();`
but the job started to be created every minute, without starting from that specific date 2016/8/8...

Sorry for my English.

**Comment by tallesl** [comment]
The library is behaving as it should be, clarying things a little:

```cs
// this call
Schedule<CreateRideJob>().ToRunOnceAt(dateAndTime).AndEvery(1).Minutes();

// is the equivalent of doing both
Schedule<CreateRideJob>().ToRunOnceAt(dateAndTime);

// and
Schedule<CreateRideJob>().ToRunEvery(1).Minutes();
```

To make a job run once every minute but only after a specific date and time you would have to do:

```cs
var delay = (int)(dateAndTime - DateTime.Now).TotalSeconds;
Schedule<CreateRideJob>().ToRunEvery(1).Minutes().DelayFor(delay).Seconds();
```

Yeah, I know, it's not pretty.
I could implement a `DelayUntil` that accepts a `DateTime`, like:

```cs
Schedule<CreateRideJob>().ToRunEvery(1).Minutes().DelayUntil(dateAndTime);
```

**Comment by AlexAlexGoTO** [comment]
Super, thank you !!!

**Comment by ismijgelijk** [comment]
Not sure if I'm in the position to make comments
So if this is inapropriate, please do tell me.

But I also assumed that using .ToRunOnceAt()AndEvery meant that the schedule only starts at the moment passed to ToRunOnceAt.
I don't think i've seen this being documented, and can imagine that it confuses other people too.
I know it confused me, and I considered it odd behaviour.

**Comment by tallesl** [comment]
@ismijgelijk 

> Not sure if I'm in the position to make comments
So if this is inapropriate, please do tell me.

You're more than welcome to comment on issues. Sometimes people approach to me via email and I ask them to open an issue here instead, so others can comment on it.

Don't see me as the "owner" of the library, see me as the "maintainer".

> But I also assumed that using .ToRunOnceAt()AndEvery meant that the schedule only starts at the moment passed to ToRunOnceAt.

I was eager to give a feedback to @AlexAlexGoTO and I didn't realize that is really odd to think `AndEvery` separated from `ToRunOnceAt`. I agree with you.

I already have a fix for it in [another branch](https://github.com/fluentscheduler/FluentScheduler/tree/issue-86), as soon I have it tested I'll publish it on NuGet.

**Comment by tallesl** [comment]
Fixed on [5.0.0](https://github.com/fluentscheduler/FluentScheduler/releases/tag/5.0.0).
